api/app/Controllers/Http/ExplorerController.js

```diff
@@ -1159,11 +1159,7 @@ class ExplorerController {
       logger.error(`ExplorerController.command: Unknown command ${command}`);
       throw new Error(`Unknown command`);
     }
-    const ws = await this.getWorkspace();
-    const wsPath = ws.toString();
-    const configPath = path.join(wsPath, '/TFSettings.json');
-    const cfg = await this.getJsonConfig(configPath);
-    const cmdName = cfg[command] || Env.get(CONST_PATHS.commandNames[command])
+    const cmdName = Env.get(CONST_PATHS.commandNames[command])
```

Comment:
if you remove cfg[command] here then you should remove other code as well, e.g. initialisation of variable cfg and probably configPath as well as parameters of TFSettings are not required here anymore